### PR TITLE
Added test.only and it.only

### PR DESCRIPTION
**Pull request by mrduarte7**

### CODE REVIEW

File: lib/rules/no-only-experiments.js [hunk 1]
```diff
@@ -11,7 +11,7 @@
 module.exports = {
     meta: {
         docs: {
-            description: "Disallow .only for experiments and describes",
+            description: "Disallow .only for experiments, test and describes",
```
Comment (kruzty) [inline lib/rules/no-only-experiments.js]: There is a comment at the top of the file (line 2) that should be updated.

File: tests/lib/rules/no-only-experiments.js [hunk 2]
```diff
@@ -22,8 +22,11 @@ ruleTester.run("no-only-experiments", rule, {
```
Comment (kruzty) [inline tests/lib/rules/no-only-experiments.js:22]: There is a comment at the top of the file (line 2) that should be updated.

**Review by kruzty — APPROVED** [review]

File: lib/rules/no-only-experiments.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ module.exports = {
         //----------------------------------------------------------------------
 
         function ifExperimentReport(name, node) {
-            if(name == 'experiment' || name === 'describe') {
+            if(name == 'experiment' || name === 'describe'|| name === 'test' ) {
```
Comment (jamelling) [inline lib/rules/no-only-experiments.js]: Lab has `it` as an alias of `test` similar to how `describe` is an alias of `experiment`.

File: lib/rules/no-only-experiments.js [hunk 4]
```diff
@@ -46,12 +46,12 @@ module.exports = {
                 }
 
                 if (node.object.name) {
-                    // covers the experiment.only() case
+                    // covers the experiment.only() and test.only()case
```
Comment (jamelling) [inline lib/rules/no-only-experiments.js]: You are missing a space before case

File: tests/lib/rules/no-only-experiments.js [hunk 5]
```diff
@@ -22,8 +22,11 @@ ruleTester.run("no-only-experiments", rule, {
 
     valid: [
         'lab.experiment("valid", function() {})',
+        'lab.test("valid", function() {})',
         'lab.experiment',
+        'lab.test',
         'experiment("valid", function() {})',
+        'lab("valid", function() {})',
```
Comment (jamelling) [inline tests/lib/rules/no-only-experiments.js]: You should also add a case for `lab.it('valid', function() {})` and `lab.it`

Comment (mrduarte7) [reply tests/lib/rules/no-only-experiments.js]: Okay can do! They didn't add a case for describe so I just followed what they had done. So I should also add a case for describe as well.

Comment (mrduarte7) [reply tests/lib/rules/no-only-experiments.js]: Would there need to be a test case for 'it("valid", function() {})' ?

Comment (jamelling) [reply tests/lib/rules/no-only-experiments.js]: That seems like a good idea

**Review by jamelling — APPROVED** [review]

**Pull request merged**